.rpsrc

```diff
@@ -11,7 +11,10 @@
   "cors_headers": "Origin, X-Requested-With, Accept, Content-Type, csrf-token, authorization",
   "cors_methods": "*",
   "mps_server": "http://localhost:3000",
-  "delay_timer": 12,
+  "delay_activation_sync": 12000,
```

Comment:
Changing delay_timer name is a breaking change.  Also, moving from seconds to milliseconds in the value units is also a breaking change

Reply:
fixed (put it back and only exposed delay_timer in .rpsrc file)